### DISCUSSION TITLE
How to use host browser when click a link in sandboxed app?

### DISCUSSION THREAD

**Discussion by GF-Huang** [sandboxie-plus/Sandboxie#3550] [Documents and notes]
In default, it will launch the sandboxed browser.

**Comment by hg421** [comment]
For one time only or always? If always, setting the browser as a breakout program should work.

**Reply by GF-Huang** [reply]
Always, and how to do that?

**Reply by hg421** [reply]
go to Sandbox Options -> Program Control -> Breakout Programs

**Reply by GF-Huang** [reply]
I found it, thanks.

![image](https://github.com/sandboxie-plus/Sandboxie/assets/4510984/3414dddf-28d7-48b5-9545-19fcb4e2d916)

**Reply by GF-Huang** [reply]
And if for one time, how to do that?

**Reply by hg421** [reply]
> And if for one time, how to do that?

I don't know if there is a better way, but you could temporarily set the breakout option and then disable it again (using the checkmark)
